### DISCUSSION TITLE
System.ServiceProcess.ServiceController.GetServices(). Different result.

### DISCUSSION THREAD

**Discussion by SanyaSa** [dotnet/runtime#82296] [Q&A]
The same code works differently. If you build under framework4.8 the result is different from NET6. Is that how it should be?

**Comment by danmoseley** [comment]
How is it different?

**Reply by SanyaSa** [reply]
Do you know what the problem is? And when will it be corrected? Or is that how it's meant to be?

**Comment by SanyaSa** [comment]
![0](https://user-images.githubusercontent.com/17914425/219969424-5801ae2d-5fba-4244-af08-20fbaf3c823c.png)
![1](https://user-images.githubusercontent.com/17914425/219969806-7035ebfe-3c47-43d9-9eb6-9f844f47f5f8.png)

If you build it under Framework 4.8, it will return several services. If you build it under NET.6, it does not return anything. But the most interesting thing is that the request is sent from Windows OS to the server where Linux OS is installed.
